webapp/app.py

```diff
@@ -1,38 +1,23 @@
 import flask
-import talisker.flask
-import talisker.logs
-from werkzeug.debug import DebuggedApplication
-from werkzeug.middleware.proxy_fix import ProxyFix
 
-from webapp.extensions import sentry
+from canonicalwebteam.flask_base.app import FlaskBase
 from webapp.external_urls import external_urls
-from webapp.handlers import add_headers, clear_trailing_slash
+from webapp.handlers import add_headers
 from webapp.jaasai.views import jaasai
 from webapp.redirects.views import jaasredirects
 from webapp.store.views import jaasstore
 from webapp.template_utils import current_url_with_query, static_url
 
 
 def create_app(testing=False):
-    app = flask.Flask(
-        __name__, template_folder="../templates", static_folder="../static"
+    app = FlaskBase(
```

Comment:
You can now pass the 404 and 500 templates in here and flask-base registers the error handlers.

Reply:
See https://github.com/canonical-web-and-design/canonical.com/blob/f57b78ec22dad899eb527334f5552921760fc81d/webapp/app.py#L10